plugins/outputs/cloudwatchlogs/pusher.go

```diff
@@ -84,15 +84,17 @@ func NewPusher(target Target, service CloudWatchLogsService, flushTimeout time.D
 
 func (p *pusher) AddEvent(e logs.LogEvent) {
 	if !hasValidTime(e) {
-		p.Log.Errorf("The log entry in (%v/%v) with timestamp (%v) comparing to the current time (%v) is out of accepted time range. Discard the log entry.", p.Group, p.Stream, e.Time(), time.Now())
+		p.Log.Errorf("Discarding log entry because entry is out of accepted time range")
```

Comment:
I spoke with @ZhenyuTan-amz, and he wanted to go with this instead of having an if statement that checked if error log has already been seen since it will require using contains which will add an o(n) computation each time invalid time is hit.

Reply:
Spoke with @SaxyPandaBear and he did not want us to use this behavior since it does not fix the issue but instead pushes it to debug only.